test/perf/flatMap.js

```diff
@@ -44,11 +45,16 @@ suite
 	.add('most', function(deferred) {
 		runners.runMost(deferred, most.from(a).flatMap(most.from).reduce(sum, 0));
 	}, options)
-	.add('rx', function(deferred) {
+	.add('rx 4', function(deferred) {
 		runners.runRx(deferred, rx.Observable.fromArray(a).flatMap(rx.Observable.fromArray).reduce(sum, 0));
 	}, options)
+	.add('rx 5', function(deferred) {
+		runners.runRx5(deferred,
+			rxjs.Observable.fromArray(a).flatMap(
+				function(x) {return rxjs.Observable.fromArray(x)}).reduce(sum, 0))
+	}, options)
```

Comment:
This caught my eye just now.  Does rx5 require that static combinators like `fromArray` are called with `rxjs.Observable` as it's `this` arg?  If so, I wonder why they've made that decision.